aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +261,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Forms in layers vs. full page 
+
+
+- Single column form layout creates a clear focused path to completion for the user.
+- Multi-column layouts can cause users to skip important fields, input data in a wrong field, or simply hinder efficiency which leads to frustration or even abandonment. 
+- Display no more than 10 form fields with in the same viewport so that users are not intimidated with seeing a long page filled with form fields and sections.
+- Use a [Center modals](https://design-system.hpe.design/components/layer#center-modal) instead of side drawer to display information without leaving the current contexts. 
```

Comment:
I am still unsure as to what this is telling me / why this is being recommended. Is there a visual someone could share?

This reads as if I am never supposed to use a side drawer for a form. Under which circumstances should I use a side drawer vs. full page vs. center modal? Perhaps this gets clearer one the "forms in layer vs. full-page" content is written?

Reply:
content was changed